### PR TITLE
fix daemon controller runtime error

### PR DESCRIPTION
**Pull request by hhyasdf**
Pull Request Description
---

### Special notes for reviews

### CODE REVIEW

File: cmd/daemon/daemon.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func main() {
 	entryLog.Info("starting hybridnet daemon",
 		"known-features", feature.KnownFeatures(), "commit-id", gitCommit)
 
-	config, err := daemonconfig.ParseFlags()
+	config, err := daemonconfig.ParseFlags(entryLog)
```
Comment (mars1024) [inline cmd/daemon/daemon.go:46]: no need to take logger as an input, we can log error and config outside

Comment (hhyasdf) [reply cmd/daemon/daemon.go]: Yes, I've found that too. Fixed.

File: pkg/daemon/controller/utils.go [hunk 2]
```diff
@@ -55,7 +58,10 @@ func (t *simpleTriggerSource) Start(ctx context.Context, handler handler.EventHa
 }
 
 func (t *simpleTriggerSource) Trigger() {
-	t.queue.Add(t.key)
+	t.queue.Add(reconcile.Request{NamespacedName: types.NamespacedName{
+		Name:      t.key,
+		Namespace: "All",
```
[the remark this answers is not in the document]
Comment (hhyasdf) [reply pkg/daemon/controller/utils.go]: Fixed.

**Review by mars1024 — APPROVED** [review]
/lgtm

**Pull request merged**